components/02-molecules/pull-quote/_pull-quote.scss

```diff
@@ -0,0 +1,59 @@
+:root {
+  --color-gray-light: #{$gray-light};
+  --color-gray-lightest: #{$gray-lightest};
+  --color-blue-light: #{$blue-light};
+  --color-black: #{$black};
+}
+
+.pull-quote {
+  --color-pull-quote-quote: var(--color-gray-lightest);
+  --color-pull-quote-attribution: var(--color-gray-lightest);
+  --color-pull-quote-accent: var(--color-blue-light);
+
+  background-color: var(--color-black);
+  padding: 3rem 2rem;
```

Comment:
`$space-triple` and `$space-double` variables. All other `rem` instances in this file should also use `$space`